src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -196,4 +198,22 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
                 By.cssSelector("input[id$='OptionText-" + qnNumber + "-" + responseNumber + "']"));
         waitForElementToBeClickable(element);
     }
+    
+    // ------------- For InstructorEditStudentFeedbackPage -------------
+    
+    public boolean clickModerationHintButton() {
+        click(By.id("moderationHintButton"));
+        // Check if links toggle properly.
+        WebElement moderationHint = browser.driver.findElement(By.id("moderationHint"));
+        return moderationHint.isDisplayed();
```

Comment:
I am trying to follow the style in `InstructorFeedbackResultsPageUiTest`, it test the similar situation where the panel is toggled and the text box changed.
``` java
assertEquals("[more]", resultsPage.getQuestionAdditionalInfoButtonText(8, "section-1-giver-1-recipient-1"));
        assertTrue(resultsPage.clickQuestionAdditionalInfoButton(8, "section-1-giver-1-recipient-1"));
        assertEquals("[less]", resultsPage.getQuestionAdditionalInfoButtonText(8, "section-1-giver-1-recipient-1"));
        assertFalse(resultsPage.clickQuestionAdditionalInfoButton(8, "section-1-giver-1-recipient-1"));
        assertEquals("[more]", resultsPage.getQuestionAdditionalInfoButtonText(8, "section-1-giver-1-recipient-1"));
```
If this is not acceptable, we need another issue and PR to make the test case `InstructorFeedbackResultsPageUiTest` better as well.